src/main/java/com/minecolonies/colony/Schematics.java

```diff
@@ -20,16 +24,30 @@
  */
 public final class Schematics
 {
-    private static final String                    NULL_STYLE            = "schematics";
-    private static final String                    SCHEMATIC_EXTENSION   = ".schematic";
-    private static final String                    SCHEMATICS_ASSET_PATH = "/assets/minecolonies/schematics/";
-    //Hut, Styles
-    private static       Map<String, List<String>> hutStyleMap           = new HashMap<>();
-    //Hut, Levels
-    @NotNull
-    private static       Map<String, Integer>      hutLevelsMap          = new HashMap<>();
-    //Decoration, Style
-    private static       Map<String, List<String>> decorationStyleMap    = new HashMap<>();
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)